.github/workflows/test.yml

```diff
@@ -144,7 +144,7 @@ jobs:
       # - name: Setup tmate session
       #   uses: mxschmitt/action-tmate@v3
 
-      - run: pnpm run jest integration/${{ matrix.database }} -- --forceExit --maxConcurrency=8
+      - run: pnpm run jest integration/${{ matrix.database }} -- --forceExit --maxConcurrency=8 --verbose
```

Comment:
We can probably remove this
```suggestion
      - run: pnpm run jest integration/${{ matrix.database }} -- --forceExit --maxConcurrency=8
```

Reply:
I added it to see which tests we were running. Only with that it was possible to see that this was only running 1 test vs. many as expected.

Reply:
Since we removed `slice(0,1)` I don't think we need it though?

Reply:
It does help indeed